kb/temporal-platform-limits-sheet.md

```diff
@@ -32,7 +31,8 @@ Unless otherwise stated, Event limits apply to all Events occurring simultaneous
   - We warn at 10,000 Events: `history size exceeds warn limit.`
   - We error at 50,000 Events: [`history size exceeds error limit.`](https://github.com/temporalio/temporal/blob/v1.7.0/service/history/workflowExecutionContext.go#L1204)
   - This is configurable with [`HistoryCountLimitError` and `HistoryCountLimitWarn`](https://github.com/temporalio/temporal/blob/v1.7.0/service/history/configs/config.go#L382-L383), if you know what you are doing.
-  - We fail the following Commands at 50,000 Events for OSS and 2,000 Events for Cloud:
+- **Concurrent Action limit**
+  - We fail the following Commands at 50,000 Events for OSS and 2,000 concurrent Actions for Cloud:
```

Comment:
We fail the following action Commands if concurrent running action count exceeds limit. The limit on cloud is 2000. There is no default limit on OSS server, but can ben enforced by dynamic config...